addons/calendar/views/calendar_views.xml

```diff
@@ -332,48 +332,65 @@
-                            <field name="videocall_location" string="Meeting Link" widget="CopyClipboardChar" force_save="1" readonly="videocall_source == 'discuss'"/>
-                            <button name="clear_videocall_location" type="object" class="btn btn-link p-0"
-                                invisible="not videocall_location" context="{'recurrence_update': recurrence_update}">
-                                <span class="fa fa-times"></span><span> Clear meeting</span>
-                            </button>
-                            <button name="set_discuss_videocall_location" type="object" class="btn btn-link p-0"
-                                invisible="videocall_location" context="{'recurrence_update': recurrence_update}">
-                                <span class="fa fa-plus"></span> <span>Odoo meeting</span>
-                            </button>
-                            <button name="action_join_video_call" class="btn btn-link" help="Join Video Call" type="object" invisible="not videocall_location">
-                                <span class="oi oi-arrow-right"></span><span> Join video call</span>
-                            </button>
-                        </div>
-                        <field name="description" placeholder="Describe your meeting"/>
-                    </group>
-                </sheet>
+                    <field name="access_token" force_save="1" invisible="1"/>
+                    <field name="videocall_location" force_save="1" invisible="1"/>
+                    <field name="videocall_source" force_save="1" invisible="1"/>
+                    <button name="set_discuss_videocall_location" type="object" class="mx-1 text-nowrap"
+                        invisible="videocall_location" context="{'recurrence_update': recurrence_update}">
+                        <span class="fa fa-plus me-1"/>Add Video
+                    </button>
+                </div>
+                <div class="d-flex align-items-center" colspan="2" invisible="not videocall_location">
+                    <i class="fa fa-video-camera text-600" title="Videocall URL"/>
+                    <field name="videocall_location" nolabel="1"
+                        widget="CalendarLocationCopyClipboardChar"
+                        class="w-100 mb-0 o_field_CopyClipboardChar" 
+                        readonly="0"
+                        force_save="1"
+                    />
+                </div>
+                <div class="d-flex align-items-center" colspan="2">
+                    <i class="fa fa-lock text-600" title="Privacy"/>
+                    <field name="privacy" nolabel="1" class="w-100 mb-0" placeholder="Privacy"/>
+                </div>
+                <div class="d-flex" colspan="2">
+                    <i class="fa fa-sticky-note mt-2 text-600" title="Notes"/> 
+                    <field name="description" nolabel="1" class="w-100" placeholder="Notes" widget="calendar_event_notes_html"/>
```

Comment:
A bit confusing. Notes is the new field. I do not know if we want to display the `notes` fields (not sync'd with external calendars) or the `description` here. Probably the notes, if the AMDI task computes the description automatically.

Reply:
(Notes at creation, and description on the popup? probably worth checking with bejo. Could change with AMDI task)